Examples/ObjcExample/ObjcExample.xcodeproj/project.pbxproj

```diff
@@ -181,7 +181,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "source ~/.bash_profile\ncd ${SRCROOT}\ncd ..\nclang-format -style=file -i ObjcExample/**/*.m ObjcExample/**/*.h\n";
+			shellScript = "source ~/.bash_profile\ncd ${SRCROOT}\ncd ../..\nmake install-examples\n\nclang-format -style=file -i Examples/ObjcExample/**/*.m Examples/ObjcExample/**/*.h\n";
```

Comment:
This script does not seem to prevent us from manually running `pod install --project-directory=Examples` before trying to compile. You can seen the problem when trying to run this from a clean state.

Reply:
Yes, it does not alleviate the need for a first `pod install --project-directory=Examples`. I could not get that work, but I didn't really delve deep into it. This will automatically install VelocidiSDK if any changes were made to it after that.

Reply:
Ok. But then this is kinda misleading: the `install` part does nothing for the current compilation session. 

Wouldn't we be better off without `make install-examples` in this script?

Reply:
> Ok. But then this is kinda misleading: the `install` part does nothing for the current compilation session.

Yes, that is true. I could not find a way to add a script that would run before the `Check Dependencies` phase, that is why this fails to avoid that.

> Wouldn't we be better off without `make install-examples` in this script?

I find it useful to have any change to VelocidiSDK be automatically added when running one of the examples project, hence why I added this. 

I can add a comment stating that `make install-examples` is just being executed to automatically include any change made in VelocidiSDK.

Reply:
> I find it useful to have any change to VelocidiSDK be automatically added when running one of the examples project, hence why I added this.

Sure, but doesn't that usefulness come from having the example picking up the most recent changes?

From what I understand, with the current modifications on this PR, the version of the `VelocidiSDK` that is installed when you compile one of the examples won't be the version that the example is running against. So, you might be running an example expecting to see the behavior of some change you made inside `VelocidiSDK` but that won't happen... or am I missing something?

Reply:
You're right, this wasn't working as intended. I tested this by changing the content of the User-Agent sent, and it worked, but stuff that is verified on compilation (like linking methods), will not work and will throw an error. Removed in 5b154e4